package.json

```diff
@@ -16,6 +16,9 @@
     "url": "https://github.com/PubMatic/OpenWrap/issues"
   },
   "homepage": "https://github.com/PubMatic/OpenWrap#readme",
+  "dependencies": {
+    "uas-adclient": "0.0.1-master.5"
```

Comment:
what happens when someone who does not have access to uas-adclient repo checks out the OpenWrap code and tries to build ?